test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,70 @@
+require('dotenv').config();
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+
+chai.use(chaiSubset);
+
+const urlBase = 'https://api.github.com';
+const githubUserName = 'Ktagudelo';
+
+const searchGist = async () => {
+  const response = await axios.get(`${urlBase}/users/${githubUserName}/gists`, {
+    headers: {
+      Authorization: `token ${process.env.ACCESS_TOKEN}`
+    }
+  });
+
+  const search = response.data.find(
+    (element) => element.description === 'Example of a gist'
+  );
+  return search.id;
+};
```

Comment:
if you store the response of the call when you create the gist, you don't need this function, since the response data will contain the id of the gist